src/app/level2/level2.component.html

```diff
@@ -10,45 +10,81 @@ <h1>Level 2</h1>
   <tbody>
     <tr>
       <td class="indicator">
-        <app-column-indicator [columnIndex]="0" (drop)="drop($event)" [playerIndex]="currentPlayerIndex"></app-column-indicator>
+        <app-column-indicator [columnIndex]="0" (drop)="drop($event)" [playerIndex]="currentPlayerIndex">
+        </app-column-indicator>
       </td>
       <td class="indicator">
-        <app-column-indicator [columnIndex]="1" (drop)="drop($event)" [playerIndex]="currentPlayerIndex"></app-column-indicator>
+        <app-column-indicator [columnIndex]="1" (drop)="drop($event)" [playerIndex]="currentPlayerIndex">
+        </app-column-indicator>
       </td>
       <td class="indicator">
-        <app-column-indicator [columnIndex]="2" (drop)="drop($event)" [playerIndex]="currentPlayerIndex"></app-column-indicator>
+        <app-column-indicator [columnIndex]="2" (drop)="drop($event)" [playerIndex]="currentPlayerIndex">
+        </app-column-indicator>
       </td>
       <td class="indicator">
-        <app-column-indicator [columnIndex]="3" (drop)="drop($event)" [playerIndex]="currentPlayerIndex"></app-column-indicator>
+        <app-column-indicator [columnIndex]="3" (drop)="drop($event)" [playerIndex]="currentPlayerIndex">
+        </app-column-indicator>
       </td>
     </tr>
     <tr>
-      <td><div class="coin"></div></td>
-      <td><div class="coin"></div></td>
-      <td><div class="coin"></div></td>
-      <td><div class="coin"></div></td>
+      <td>
+        <div class="coin" [class] = "getStyle(0,0)"></div>
```

Comment:
Minor: Formatting is a little bit unusual for HTML (no space before/after `=`). Format document before handing it in is recommended.